lib/Doctrine/Migration/Builder.php

```diff
@@ -75,7 +75,7 @@ public function __construct($migrationsPath = null)
         if ($migrationsPath instanceof Doctrine_Migration) {
             $this->setMigrationsPath($migrationsPath->getMigrationClassesDirectory());
             $this->migration = $migrationsPath;
-        } else if (is_dir($migrationsPath)) {
+        } else if (is_dir((string) $migrationsPath)) {
```

Comment:
This is also pretty common. Many core functions throw warnings if they expect a string argument and they get passed a `null`. Lots of situations where that's possible now have the argument being cast to `string` to avoid those warnings.